packages/app/src/cli/services/context/partner-account-info.ts

```diff
@@ -2,37 +2,9 @@ import {getCurrentAccountInfo} from '../../api/graphql/current_account_info.js'
 import {getCachedAccountInfo, setCachedAccountInfo} from '../../utilities/app-conf-store.js'
 import {DeveloperPlatformClient} from '../../utilities/developer-platform-client.js'
 import {outputDebug} from '@shopify/cli-kit/node/output'
+import {AccountInfo, isUserAccount, isServiceAccount} from '@shopify/cli-kit/node/session'
 
-export interface PartnersSession {
-  token: string
-  businessPlatformToken: string
-  accountInfo: AccountInfo
-  userId: string
-}
-
-export type AccountInfo = UserAccountInfo | ServiceAccountInfo | UnknownAccountInfo
-
-interface UserAccountInfo {
-  type: 'UserAccount'
-  email: string
-}
-
-interface ServiceAccountInfo {
-  type: 'ServiceAccount'
-  orgName: string
-}
-
-interface UnknownAccountInfo {
-  type: 'UnknownAccount'
-}
-
-export function isUserAccount(account: AccountInfo): account is UserAccountInfo {
-  return account.type === 'UserAccount'
-}
-
-export function isServiceAccount(account: AccountInfo): account is ServiceAccountInfo {
-  return account.type === 'ServiceAccount'
-}
+export {AccountInfo, isUserAccount, isServiceAccount}
```

Comment:
No need to re-export them here no? Whoever is using these can directly import from cli-kit